public/components/trace_analytics/components/common/shared_components/component_helper_functions.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { useEffect } from 'react';
+
+// Injects the size warning and Load buttons into the corners of EUI Data Grid
+export const useInjectElementsIntoGrid = (
+  rowCount: number,
+  maxDisplayRows: number,
+  tracesTableMode: string,
+  loadMoreHandler?: () => void
+) => {
+  useEffect(() => {
+    setTimeout(() => {
+      const toolbar = document.querySelector<HTMLElement>('.euiDataGrid__controls');
+      if (toolbar && rowCount > maxDisplayRows) {
+        toolbar.style.display = 'flex';
+        toolbar.style.alignItems = 'center';
+        toolbar.style.justifyContent = 'space-between';
+
+        let warningDiv = toolbar.querySelector<HTMLElement>('.trace-table-warning');
+        if (!warningDiv) {
+          warningDiv = document.createElement('div');
```

Comment:
Can we  use scss with a custom class for div and then we can use OUI variables `$ouiColorWarning`

Reply:
Now using
```
.trace-table-warning {
  margin-left: auto;
  color: $euiColorWarning;
  font-size: $euiFontSizeS;
  display: flex;
  align-items: center;
  padding: $euiSizeXS $euiSize;
  border-radius: $euiBorderRadius;
  white-space: nowrap;
}
```